site/src/Librecores/ProjectRepoBundle/Entity/SourceRepo.php

```diff
@@ -102,6 +102,46 @@
     protected $sourceStats;
 
     /**
+     * Forks of this repository in the repository host
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $forks;
+
+    /**
+     * Forks of this repository in the repository host
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $openPullRequests;
+
+    /**
+     * Forks of this repository in the repository host
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $openIssues;
+
+    /**
+     * Stars of this repository in the repository host
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $stars;
+
+    /**
+     * Watchers of this repository in the repository host
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $watchers;
```

Comment:
GitHub blurs the distinction between a project and a repository. We can still consider it to a project